typescript/cli/src/commands/core.ts

```diff
@@ -137,6 +136,16 @@ export const read: CommandModuleWithContext<{
     ),
   },
   handler: async ({ context, chain, mailbox, config: configFilePath }) => {
+    if (!mailbox) {
```

Comment:
I'm nitpicking but the structure I initially intended for these command defs was for the code under /src/commands to just dumb proxies to implementations elsewhere. My reasoning was that `yargs` is not that great and we may want to swap it out eventually. I used yargs because we were already using it elsewhere.

Reply:
given that this PR didnt introduce this structure would prefer not to change it here